src/simulator/BootGUI.java

```diff
@@ -118,73 +118,74 @@ public BootGUI(Computer computer, String[] devices)
 		memoryImage="";
 		datapathxml="";
 		controlxml="";
-		
-		try
-		{
-			FileReader fr=new FileReader("settings.txt");
-			Scanner scan=new Scanner(fr);
-			while(true)
-			{
-				if (!scan.hasNext())
-					break;
-				String type=scan.next();
-				if (type.equals("DiskA"))
-				{
-					diskIncluded[0]=true;
-					diskImage[0]=scan.next();
-				}
-				if (type.equals("DiskB"))
-				{
-					diskIncluded[1]=true;
-					diskImage[1]=scan.next();
-				}
-				if (type.equals("DiskC"))
-				{
-					diskIncluded[2]=true;
-					diskImage[2]=scan.next();
-					isCD[2]=scan.nextInt()==1;
-					cylinders[2]=scan.nextInt();
-					heads[2]=scan.nextInt();
-					sectors[2]=scan.nextInt();
-				}
-				if (type.equals("DiskD"))
-				{
-					diskIncluded[3]=true;
-					diskImage[3]=scan.next();
-					isCD[3]=scan.nextInt()==1;
-					cylinders[3]=scan.nextInt();
-					heads[3]=scan.nextInt();
-					sectors[3]=scan.nextInt();
-				}
-				if (type.equals("ROM"))
-				{
-					romImage=scan.next();
-				}
-				if (type.equals("VideoROM"))
-				{
-					vromImage=scan.next();
-				}
-				if (type.equals("MemoryContents"))
-				{
-					memoryImage=scan.next();
-					memoryImageStart=Integer.parseInt(scan.next(),16);
-				}
-				if (type.equals("CustomProcessor"))
-				{
-					try{
-					datapathxml=scan.next();
-					controlxml=scan.next();
-					}catch(java.util.NoSuchElementException e){}
-				}
-			}
-		}
-		catch(Exception e)
-		{
-			e.printStackTrace();
-		}
+
+        try
+        {
+            FileReader fr = new FileReader("settings.txt");
+            Scanner scan = new Scanner(fr);
+            while(scan.hasNext())
+            {
+                findBootType(scan);
+            }
+        }
+
+        catch (java.io.FileNotFoundException e) {
+            System.out.println("Error: Settings.txt not found");
+        }
+
```

Comment:
I don't know what other exceptions were being caught here, there may have been some though.  You can the catch(Exception e) that was here back....but AFTER your error.
This brings up another point...should we introduce a logger instead of System.out?  Maybe as another ticket.